src/models/user.js

```diff
@@ -0,0 +1,72 @@
+const mongoose = require('mongoose');
+
+const organizationSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    address: {
+      type: String,
+      required: true,
+    },
+  },
+  { timestamps: true }
+);
+
+const volunteerSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    age: {
```

Comment:
in my opinion we could replace `age` with `dateOfBirth`, its more dynamic to update user age with time, then we can set a virtual field to calculate the age.
```
birthday: { 
type: Date,
required: true
}
```